src/Process/SubdivisionControl.js

```diff
@@ -0,0 +1,27 @@
+export default {
+    preUpdate: (context, layer) => {
+        context.colorLayers = context.view.getLayers(
+            (l, a) => a && a.id == layer.id && l.type == 'color');
+        context.elevationLayers = context.view.getLayers(
+            (l, a) => a && a.id == layer.id && l.type == 'elevation');
+    },
+
+    hasEnoughTexturesToSubdivide: (context, layer, node) => {
+        // Prevent subdivision if node is covered by at least one elevation layer
+        // and if node doesn't have a elevation texture yet.
+        for (const e of context.elevationLayers) {
+            if (e.tileInsideLimit(node, e) && !node.isElevationLayerLoaded()) {
+                return false;
+            }
+        }
+
+        // Prevent subdivision if missing color texture
```

Comment:
Currently I'm requiring that all layers have a texture. Maybe we can adjust this in the future or split the conditions in 2 separate functions.